near-contract-standards/src/fungible_token/storage_impl.rs

```diff
@@ -80,7 +80,7 @@ impl StorageManagement for FungibleToken {
                 _ => storage_balance,
             }
         } else {
-            env::panic(b"The account is not registered");
+            env::panic(format!("The account {} is not registered", &env::predecessor_account_id()).as_bytes());
```

Comment:
```suggestion
            log!("The account {} is not registered", &env::predecessor_account_id());
```

Reply:
This should not be `log!` but `env::panic`